front-end/src/TermsOfService.js

```diff
@@ -107,8 +107,9 @@ function TermsOfService() {
 		<ul>
 		    <li>By visiting <Link to="/meet-the-team">our contact page.</Link></li>
 		</ul>
+		<br></br>
```

Comment:
A small note--the `br` tag can be self closing, so I think this code could look a little neater as `<br />` rather that `<br></br>`. Other than that, looks good, thanks Lev!

Reply:
I'll make that change quickly